src/main/resources/application.conf

```diff
@@ -0,0 +1,21 @@
+s3 {
+  cleanBucket = "tdr-upload-files-intg"
+  cleanBucket = ${?CLEAN_BUCKET}
+  outputBucket = "tdr-consignment-export-intg"
+  outputBucket = ${?OUTPUT_BUCKET}
+  endpoint = "https://s3.eu-west-2.amazonaws.com/"
+}
+api {
+  url = "https://api.tdr-integration.nationalarchives.gov.uk/graphql"
+  url = ${?API_URL}
+}
+auth {
+  url = "https://auth.tdr-integration.nationalarchives.gov.uk/auth"
+  url = ${?AUTH_URL}
+  clientId = "tdr-backend-checks"
+  clientSecret = ${CLIENT_SECRET}
+}
+efs {
+  rootLocation = "/tmp/export"
```

Comment:
Is it OK to use `/tmp`? It won't get cleaned up by some process when we're not expecting it?

Reply:
It almost certainly won't get overwritten because docker containers are supposed to have the process you specify and no others but it's probably better to change it to something new.

Also, this container was running as root which is shouldn't be. I've updated it to actually use the new user and changed the rootLocation to be `/home/consignment-export/export` which means we don't have to go around changing permissions on folders because the new user already has access to their home directory.

There's a small PR here for the terraform change to change the volume if you get a sec.
https://github.com/nationalarchives/tdr-terraform-modules/pull/73

Reply:
✅